api_tests/lib/ethereum.ts

```diff
@@ -52,26 +44,23 @@ async function erc20Balance(
         data: payload,
     };
 
-    const hexBalance = await web3Client.eth.call(tx);
-    return utils.toBN(hexBalance);
+    const transactionReceipt = await ethersClient.call(tx);
+    return ethers.utils.bigNumberify(transactionReceipt);
 }
 
 async function mintErc20Tokens(
     ownerWallet: EthereumWallet,
     contractAddress: string,
     toAddress: string,
-    amount: BN | string | number
+    amount: BigNumber | string | number
 ) {
     const functionIdentifier = "40c10f19";
 
     toAddress = toAddress.replace(/^0x/, "").padStart(64, "0");
```

Comment:
I wonder if the new library gives you tools to make this easier.

Reply:
For this in particular I did not find anything obvious.